TimeLog.Api.Documentation/Source/Reporting/GetEmployeesRaw.xsd

```diff
@@ -42,6 +42,7 @@
       <xsd:element name="EmployeeNo" type="xsd:string" minOccurs="0" maxOccurs="1" />
       <xsd:element name="Username" type="xsd:string" minOccurs="0" maxOccurs="1" />
       <xsd:element name="CostPrice" type="xsd:decimal" minOccurs="0" maxOccurs="1" />
+      <xsd:element name="LegalEntityID" type="xsd:positiveInteger" />

```

Comment:
Please add `minOccurs` and `maxOccurs` for completeness.

Reply:
It's added.